File: _CodeTopics/LeetCode_contest/biweekly/biweekly2021/47/47_2.py
class Solution(object):
    def checkPowersOfThree(self, n):
        """
        :type n: int
        :rtype: bool
        """
        
        while n > 0:
            remainder = n % 3
            if remainder == 2:
                return False
            n /= 3
        return True
    
"""
https://leetcode-cn.com/submissions/detail/151900755/

128 / 128 个通过测试用例
状态：通过
执行用时: 24 ms
内存消耗: 12.9 MB
"""

File: _CodeTopics/LeetCode/1601-1800/001780--biweekly47_2/tran/47_2.cpp
class Solution {
public:
    bool checkPowersOfThree(int n) {

        // 第 47 场双周赛第二题
        // https://leetcode.cn/submissions/detail/151900755/

        while (n > 0) {
            int remainder = n % 3;
            if (remainder == 2) {
                return false;
            }
            n /= 3;
        }
        return true;
    }
};

/*
https://leetcode.cn/submissions/detail/388001448/

执行用时：
0 ms
, 在所有 C++ 提交中击败了
100.00%
的用户
内存消耗：
5.8 MB
, 在所有 C++ 提交中击败了
47.14%
的用户
通过测试用例：
129 / 129
*/
